libs/sql-schema-describer/src/lib.rs

```diff
@@ -440,12 +446,21 @@ impl SqlSchema {
     }
 }
 
+#[enumflags2::bitflags]
+#[repr(u8)]
+#[derive(Clone, Copy, Debug)]
+pub enum TableProperties {
+    NoProperties,
```

Comment:
I think this variant is redundant, if 0x01 means the same thing as 0x00 (Bitflags::empty()).

Reply:
Oh yeah, you're absolutely right. This was before I understood how `Bitflags` works.